flatlaf-core/src/main/java/com/formdev/flatlaf/ui/FlatNativeLibrary.java

```diff
@@ -55,12 +55,12 @@ private static void initialize() {
 
 		String classifier;
 		String ext;
-		if( SystemInfo.isWindows_10_orLater ) {
-			// Windows: requires Windows 10/11
+		if( SystemInfo.isWindows_10_orLater && (SystemInfo.isX86 || SystemInfo.isX86_64 || SystemInfo.isAARCH64) ) {
```

Comment:
Have re-added this check for platforms to be sure to exclude unknown platforms.
E.g. Windows 10 ARM 32 bit (not sure whether Java exists for this platform).
Theoretically Microsoft could port Windows to RISC-V in the future. We don't know 😺

Reply:
AFAIK the only time a Windows variant for ARM 32bit existed (except Windows Mobile) was Windows RT, which I think is safe to say dead. But yeah, I guess it wouldn't hurt to keep the check there.